docs/examples/cloudpubsubsource/README.md

```diff
@@ -1,4 +1,104 @@
 # CloudPubSubSource Example
 
-Please refer to the [example](https://knative.dev/docs/eventing/samples/gcp-pubsub-source/) in Knative until we migrate
-it to this repo. 
+## Overview
+
+This sample shows how to configure the CloudPubSubSource. 
+This source is most useful as a bridge from other GCP services, 
+such as [Cloud Storage](https://cloud.google.com/storage/docs/pubsub-notifications), 
+[Cloud Scheduler](https://cloud.google.com/scheduler/docs/creating#).
+
+## Prerequisites
+
+1. [Install Knative with GCP](../../install/README.md).
+
+1. [Create a Pub/Sub enabled Service Account](../../install/pubsub-service-account.md)
+
+## Deployment
+
+1. Create a GCP PubSub Topic. If you change its name (`testing`), you also need
+      to update the `topic` in the
+      [`CloudPubSubSource`](cloudpubsubsource.yaml) file:
+   
+      ```shell
+      gcloud pubsub topics create testing
+      ``` 
+     
+1. Create a [`CloudPubSubSource`](cloudpubsubsource.yaml)
+ 
+     ```shell
+     kubectl apply --filename cloudpubsubsource.yaml
+     ```
+      
+1. Create a [`Service`](event-display.yaml) that the CloudAuditLogsSource will sink into:
+
+   ```shell
+   kubectl apply --filename event-display.yaml
+   ```
+   
+## Publish
+
+Publish messages to your GCP PubSub topic:
+
+```shell
+gcloud pubsub topics publish testing --message='{"Hello": "world"}'
+```
+
+## Verify
+
+We will verify that the published event was sent by looking at the logs of the
+service that this CloudPubSubSource sinks to.
+
+1. We need to wait for the downstream pods to get started and receive our event,
+   wait 60 seconds.
+
+   - You can check the status of the downstream pods with:
```

Comment:
No need for the -, I think it was an old thing as we have multiple subitems... Just add this sentence `You can check the status....` in the same paragraph as before...